PaymentKit/PTKView.m

```diff
@@ -83,7 +85,7 @@ - (void)setup
     self.backgroundColor = [UIColor clearColor];
 
     UIImageView *backgroundImageView = [[UIImageView alloc] initWithFrame:self.bounds];
-    backgroundImageView.image = [[UIImage imageNamed:@"textfield"]
+    backgroundImageView.image = [[UIImage imageNamed:@"textfield" inBundle:kPTKBundle compatibleWithTraitCollection:nil]
```

Comment:
Thank you for submitting this fix, but you'll need to handle a fallback for clients not running iOS 8:
```objc
if ([UIImage respondsToSelector:@selector(imageNamed:inBundle:compatibleWithTraitCollection:)]) {
  // ...
} else {
  NSString *filePath = [kPTKBundle pathForResource:@"textfield" ofType:@"png"];
  backgroundImageView.image = [UIImage imageWithContentsOfFile:filePath];
}
```
I'm not sure whether the project also supports building against SDKs earlier than 8.0 (up until June 1 when Apple requires it), but if so you'll need to add `__IPHONE_OS_VERSION_MAX_ALLOWED` conditional macros.

If it were me, I'd take the image-finding code and put it in a separate C function (or instance method if you prefer) to keep all the logic in one place. Either one is certainly preferred over using `#define`.